specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/DelegatedProviderOffer.json

```diff
@@ -0,0 +1,195 @@
+                    }
+                },
+                "description": "Get the specified delegated provider offer.",
+                "tags": [
+                    "DelegatedProviderOffers"
+                ],
+                "operationId": "DelegatedProviderOffers_Get",
+                "parameters": [
+                    {
+                        "$ref": "Subscriptions.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/DelegatedProviderSubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/OfferParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/DelegatedProviderOffer"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                },
+                "produces": [
+                    "application/json"
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            }
+        }
+    },
+    "definitions": {
+        "DelegatedProviderOffer": {
+            "description": "The delegated provider offer.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Properties for an delegated provider.",
+                    "$ref": "#/definitions/DelegatedProviderOfferProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "Subscriptions.json#/definitions/Resource"
+                }
+            ]
+        },
+        "DelegatedProviderOfferProperties": {
+            "description": "Properties for an delegated provider.",
+            "type": "object",
+            "properties": {
+                "subscriptionId": {
+                    "type": "string",
+                    "description": "Subscription ID of tenant using plan."
+                },
+                "usageStartTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "UTC start time for the usage bucket to which this usage aggregate belongs."
+                },
+                "usageEndTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "UTC end time for the usage bucket to which this usage aggregate belongs."
+                },
+                "instanceData": {
```

Comment:
An example for this would be great

Reply:
unfortunately, i dont have anything at the moment